src/components/Tooltips/CanvasHoverTooltip.tsx

```diff
@@ -1,27 +1,20 @@
 import { Tag } from "antd";
 import React, { PropsWithChildren, ReactElement, useCallback } from "react";
 import styled from "styled-components";
+import { useShallow } from "zustand/shallow";
 
-import {
-  AnnotationSelectionMode,
-  Dataset,
-  VECTOR_KEY_MOTION_DELTA,
-  VectorTooltipMode,
-  ViewerConfig,
-} from "../../colorizer";
+import { AnnotationSelectionMode, VECTOR_KEY_MOTION_DELTA, VectorTooltipMode } from "../../colorizer";
 import { numberToStringDecimal } from "../../colorizer/utils/math_utils";
 import { AnnotationState } from "../../colorizer/utils/react_utils";
+import { selectVectorConfigFromState } from "../../state/slices";
 import { FlexColumn, FlexRow } from "../../styles/utils";
 
+import { useViewerStateStore } from "../../state/ViewerState";
```

Comment:
I'm curious about the import organization (e.g., state-related import here vs in line 9)

Reply:
There's an import rule that's separating out imports by file capitalization, which I'm planning to disable once this series of PRs is through. Unfortunately it'll cause horrendous merge conflicts so I'm waiting to do it!